website/docs/r/policy_host_transport_node.html.markdown

```diff
@@ -0,0 +1,164 @@
+* `node_deployment_info` - (Optional)
+  * `fqdn` - (Optional) Fully qualified domain name of the fabric node.
+  * `ip_addresses` - (Required) IP Addresses of the Node, version 4 or 6.
+  * `host_credential` - (Optional) Host login credentials.
+    * `password` - (Required) The authentication password of the host node.
+    * `thumbprint` - (Optional) ESXi thumbprint or SSH key fingerprint of the host node.
+    * `username` - (Required) The username of the account on the host node.
+  * `os_type` - (Required) Hypervisor OS type. Accepted values - 'ESXI', 'RHELSERVER', 'WINDOWSSERVER', 'RHELCONTAINER', 'UBUNTUSERVER', 'HYPERV', 'CENTOSSERVER', 'CENTOSCONTAINER', 'SLESSERVER' or 'OELSERVER'.
+  * `os_version` - (Optional) Hypervisor OS version.
+  * `windows_install_location` - (Optional) Install location of Windows Server on baremetal being managed by NSX. Defaults to 'C:\Program Files\VMware\NSX\'.
+* `standard_host_switch` - (Optional) Standard host switch specification.
+  * `cpu_config` - (Optional) Enhanced Networking Stack enabled HostSwitch CPU configuration.
+    * `num_lcores` - (Required) Number of Logical cpu cores (Lcores) to be placed on a specified NUMA node.
+    * `numa_node_index` - (Required) Unique index of the Non Uniform Memory Access (NUMA) node.
+  * `host_switch_id` - (Optional) The host switch id. This ID will be used to reference a host switch.
+  * `host_switch_mode` - (Optional) Operational mode of a HostSwitch. Accepted values - 'STANDARD', 'ENS', 'ENS_INTERRUPT' or 'LEGACY'. The default value is 'STANDARD'.
+  * `host_switch_profile` - (Optional) Identifiers of host switch profiles to be associated with this host switch.
+  * `host_switch_type` - (Optional) Type of HostSwitch. Accepted values - 'NVDS' or 'VDS'. The default value is 'NVDS'.
+  * `ip_assignment` - (Required) - Specification for IPs to be used with host switch virtual tunnel endpoints. Should contain exatly one of the below:
+    * `assigned_by_dhcp` - (Optional) Enables DHCP assignment.
+    * `static_ip` - (Optional) IP assignment specification for Static IP List.
+      * `ip_addresses` - (Required) List of IPs for transport node host switch virtual tunnel endpoints.
+      * `subnet_mask` - (Required) Subnet mask.
+      * `default_gateway` - (Required) Gateway IP.
+    * `static_ip_mac` - (Optional) IP and MAC assignment specification for Static IP List.
+      * `default_gateway` - (Required) Gateway IP.
+      * `subnet_mask` - (Required) Subnet mask.
+      * `ip_mac_pair` - (Required) List of IPs and MACs for transport node host switch virtual tunnel endpoints.
+        * `ip` - (Required) IP address.
+        * `mac` - (Required) MAC address.
+        * `static_ip_pool` - (Optional) IP assignment specification for Static IP Pool.
```

Comment:
Is this policy path? Lets try to avoid confusion..

Reply:
Sure. Changed.